roles/dtc/deploy/tasks/sub_main.yml

```diff
@@ -30,12 +30,20 @@
 
 - ansible.builtin.debug: msg="Configuring NXOS Devices using NDFC (Direct to Controller)"
 
-- name: Config-Save for Fabric {{ MD.vxlan.global.name }}
-  cisco.dcnm.dcnm_rest:
-    method: POST
-    path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/{{ MD.vxlan.global.name }}/config-save"
-  when: MD_Extended.vxlan.topology.switches | length > 0
-  # TODO: Need to add logic to only save if changes are made
+- name: Config-Save block
+  block:
+    - name: Config-Save for Fabric {{ MD.vxlan.global.name }}
+      cisco.dcnm.dcnm_rest:
+        method: POST
+        path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/{{ MD.vxlan.global.name }}/config-save"
+      when: MD_Extended.vxlan.topology.switches | length > 0
+      register: config_save
+      # TODO: Need to add logic to only save if changes are made
+  
+  rescue:
```

Comment:
Is there a specific scenario that is failing for you?  I recall this use to just be an ignore errors but if we actually get a failure we are just printing the message and moving on.  Should we actually stop execution if we encounter a failure?